### PR TITLE
Version 2.16

### PR DESCRIPTION
**Pull request by enobayram**
TODO:

- [x] Confirm whether the "Implement principal namespaces" changelog entry remains
- [x] Add a link to the changelon entry for Pact 4.4
- [ ] Final confirmation of the integration tests from @ggobugi27

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+This version will expire on **2022-12-01T00:00:00Z**.
+
+To upgrade, pull the latest docker image or download the binary and restart the
+node.
+
+Changes:
+
+*   Implement principal namespaces.
+*   Upgrade to Pact 4.4. This release brings namespaced keysets, and new
+    zero-knowledge primitives.
```
Comment (larskuhtz) [inline CHANGELOG.md:18]:
```suggestion
*   Upgrade to Pact 4.4. This release brings namespaced keysets. For further changes check the [Pact Changelog](https://github.com/kadena-io/pact/blob/master/CHANGELOG.md).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+This version replaces all previous versions. Any prior version will stop working
+on **2022-09-01T00:00:00Z**. Node administrators must upgrade to this version
+before that date.
+
+This version will expire on **2022-12-01T00:00:00Z**.
+
+To upgrade, pull the latest docker image or download the binary and restart the
+node.
+
+Changes:
```
Comment (enobayram) [inline CHANGELOG.md:14]: @larskuhtz suggested that we include this link here: https://github.com/kadena-io/chainweb-node/compare/2.15...master

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,33 @@
...
+*   Implement principal namespaces.
+*   Upgrade to Pact 4.4. This release brings namespaced keysets, and new
+    zero-knowledge primitives.
+*   Improve performance during catchup.
+*   Improve P2P protocol performance.
+*   Improve TLS session management.
+*   Faster and safer binary serialization across the board.
+*   Use rocksdb-specific features to improve performance.
+*   rocksdb is now built alongside chainweb-node instead of using the system
+    rocksdb installation.
```
Comment (enobayram) [inline CHANGELOG.md:24]:
```suggestion
```

File: CHANGELOG.md (hunk 2 above)
Comment (enobayram) [inline CHANGELOG.md:14]:
```suggestion
Changes (https://github.com/kadena-io/chainweb-node/compare/2.15...master):
```

Comment (enobayram) [inline CHANGELOG.md:14]:
```suggestion
Changes (https://github.com/kadena-io/chainweb-node/compare/2.15...2.16):
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,32 @@
...
+before that date.
+
+This version will expire on **2022-12-01T00:00:00Z**.
+
+To upgrade, pull the latest docker image or download the binary and restart the
+node.
+
+Changes:
+
+*   Implement principal namespaces.
```
Comment (enobayram) [inline CHANGELOG.md:16]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,32 @@
...
+*   Upgrade to Pact 4.4. This release brings namespaced keysets. For further changes check the [Pact Changelog](https://github.com/kadena-io/pact/blob/master/CHANGELOG.md).
+*   Improve performance during catchup.
+*   Improve P2P protocol performance.
+*   Improve TLS session management.
+*   Faster and safer binary serialization across the board.
```
Comment (enobayram) [inline CHANGELOG.md:21]:
```suggestion
*   Improve performance during catchup. (?)
*   Improve P2P protocol performance. (#1502)
*   Improve TLS session management. (#1489)
*   Faster and safer binary serialization across the board. (#1494)
```

File: src/Chainweb/Version.hs [hunk 6]
```diff
@@ -995,8 +995,8 @@ chainweb216Pact aoa v h = case aoa of
     At -> go (==) v h
     After -> go (<) v h
   where
-    go f Mainnet01 = f 2988358 -- 2022-09-02 00:00:00+00:00
-    go f Testnet04 = f 2516927 -- 2022-09-01 12:00:00+00:00
+    go f Mainnet01 = f 2988326 -- 2022-09-02 00:00:00+00:00
+    go f Testnet04 = f 2516739 -- 2022-09-01 12:00:00+00:00
```
Comment (larskuhtz) [inline src/Chainweb/Version.hs:999]: Here the latest estimates:

```
|    | network   | time                      |   height |
|---:|:----------|:--------------------------|---------:|
|  0 | testnet   | 2022-09-01 12:00:00+00:00 |  2516739 |
|  1 | mainnet   | 2022-09-02 00:00:00+00:00 |  2988324 |
```

Comment (enobayram) [reply src/Chainweb/Version.hs:999]:
```suggestion
    go f Mainnet01 = f 2988324 -- 2022-09-02 00:00:00+00:00
    go f Testnet04 = f 2516739 -- 2022-09-01 12:00:00+00:00
```

File: src/Chainweb/Version.hs [hunk 7]
```diff
@@ -995,8 +995,8 @@ chainweb216Pact aoa v h = case aoa of
     At -> go (==) v h
     After -> go (<) v h
   where
-    go f Mainnet01 = f 2988358 -- 2022-09-02 00:00:00+00:00
-    go f Testnet04 = f 2516927 -- 2022-09-01 12:00:00+00:00
+    go f Mainnet01 = f 2988324 -- 2022-09-02 00:00:00+00:00
+    go f Testnet04 = f 2516739 -- 2022-09-01 12:00:00+00:00
```
Comment (larskuhtz) [inline src/Chainweb/Version.hs:999]: |    | network   | time                      |   height |
|---:|:----------|:--------------------------|---------:|
|  0 | testnet   | 2022-09-01 12:00:00+00:00 |  2516740 |
|  1 | mainnet   | 2022-09-02 00:00:00+00:00 |  2988323 |

**Review by larskuhtz — APPROVED** [review]

**Pull request merged**